eth/downloader/queue.go

```diff
@@ -67,6 +67,7 @@ type fetchResult struct {
 	Uncles       []*types.Header
 	Transactions types.Transactions
 	Receipts     types.Receipts
+	Withdrawals  types.Withdrawals
```

Comment:
So,; one thing I'm a bit confused about... we also have `les/downloader/queue.go` - which is very similar to this file. Do we use it? 
And if so -- do we need to apply the same change there?